torch_xla/csrc/torch_util.cpp

```diff
@@ -46,6 +46,15 @@ at::Tensor UnwrapNumber(const at::Tensor& tensor, at::ScalarType dtype) {
                                                            : tensor;
 }
 
+at::Tensor MaybeWrapTensorToFunctional(const at::Tensor& tensor) {
+  bool disable_functionalization =
+      xla::sys_util::GetEnvBool("DISABLE_FUNCTIONALIZATION", false);
```

Comment:
maybe prefix it with `XLA` to match our other env vars?

Reply:
Updated